dump_frames.py

```diff
@@ -162,7 +160,7 @@ def main():
     try:
         list(
             tqdm(
-                pool.imap_unordered(dump_frames_star, dump_frames_tasks),
```

Comment:
Oh man I didn't actually know starmap existed, that's awesome! However, `starmap` will block until the next ordered result is completed, whereas `imap_unordered` will update the progress bar as soon as any video's frame is finished dumping. E.g., if the first video is super long and the rest are short, the progress bar won't update until the first one is completed. Maybe we can keep the uglier imap_unordered for now, unless you know of an alternative?